### ISSUE TITLE
[Bug] `--target newest` vs `--target minor`

### ISSUE THREAD

**Issue by o-alexandrov** [raineorshine/npm-check-updates#1220]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14.14`

---

## Steps to Reproduce

1. Set any package (w/ available `2.0.9` & `2.0.10` versions) in `package.json` to `2.0.9` 
2. Run `ncu -t minor` (it will find `2.0.10`)

## Current Behavior

Run `ncu -t newest` (it will not find a new update)

## Expected Behavior

Run `ncu -t newest` (it should find `2.0.10`)

**Comment by o-alexandrov** [comment]
Thank you for the reply.
Here's the output for the package that is failing

`npm view @ORG/PKG_NAME time`
```
{
  '2.0.0': '2022-11-14T05:15:32Z',
  '2.0.1': '2022-11-14T05:19:08Z',
  '2.0.10': '2022-11-14T10:35:34Z',
  '2.0.2': '2022-11-14T05:40:52Z',
  '2.0.3': '2022-11-14T05:46:51Z',
  '2.0.4': '2022-11-14T05:55:44Z',
  '2.0.5': '2022-11-14T06:45:21Z',
  '2.0.6': '2022-11-14T07:52:26Z',
  '2.0.7': '2022-11-14T07:55:02Z',
  '2.0.8': '2022-11-14T08:28:41Z',
  '2.0.9': '2022-11-14T08:31:31Z'
}
```

**Comment by o-alexandrov** [comment]
Thank you. I'm sorry for not being more useful from my end.
This screencast might assist on this, just so there's no doubt about the inconsistency possibility

https://user-images.githubusercontent.com/9992724/201681052-3d04ecbb-7d2d-4b8b-bb07-33f656b81b52.mov

**Comment by raineorshine** [comment]
I attempted a fix in `v16.4.1`, please give it a try.

**Comment by o-alexandrov** [comment]
Perfect, thank you very much!
Upgrades from `16.3.16` to `16.4.1` had the fix.
